src/julia.h

```diff
@@ -169,12 +166,15 @@ typedef struct {
             uintptr_t gc_bits:2;
         };
     };
-    jl_value_t value;
+    // variable length
+    void *value[1];
```

Comment:
this appears to change the size of `jl_taggedvalue_t`, which isn't entirely optimal for gdb. this may be the best we can do, but so long as we are doing this, perhaps we can make it more even useful:
```
typedef union {
    uint8_t u8;
    int8_t i8;
    ...
    struct _jl_lambda_info_t linfo;
    struct _jl_expr_t expr;
    struct _jl_array_t array;
    struct _jl_svec_t svec; // pretty sure this one won't be viable, because it is variable length (same for jl_datatype_t)
} jl_value_t;
```
then a `jl_value_t*` (such as is found in `jl_breakpoint`) can be "cast" in gdb to a lambda-info (for example) by writing `v->linfo`